src/middleware/jwt-authenticate/index.js

```diff
@@ -17,13 +17,14 @@ export default (options={}) => {
   }
 
   function middleware(req, res, next) {
-    if (!req.headers || !req.headers.authorization) {
-      return next(new UnauthorizedError('Authorisation header has not been set'));
+    const authorization = Object.assign(req.body || {}, req.query || {}, req.headers || {}).authorization;
+    if (!authorization) {
+      return next(new UnauthorizedError('Authorisation has not been set'));
     }
 
-    const [schema='', token=''] = req.headers.authorization.split(' ');
+    const [schema='', token=''] = authorization.split(' ');
```

Comment:
Will this work as expected (schema and token created under `const`) or does this create two global scoped variables?